src/PerimeterxActivitiesClient.php

```diff
@@ -64,9 +64,14 @@ public function prepareActivitiesRequest($activityType, $pxCtx, $details = [])
     }
 
     public function sendToPerimeterx($activityType, $pxCtx, $details) {
-        $details['cookie_origin'] = $pxCtx->getCookieOrigin();
+        $cookieOrigin = $pxCtx->getCookieOrigin();
+        if ($cookieOrigin && $cookieOrigin != '') {
+            $details['cookie_origin'] = $cookieOrigin;
+        }
         $details['http_method'] = $pxCtx->getHttpMethod();
-
+        $details['http_version'] = $pxCtx->getHttpVersion();
+        $details['client_uuid'] = $pxCtx->getUuid();
+        $details['risk_rtt'] = $pxCtx->getRiskRtt();
```

Comment:
according to what's written right now in the spec - if a risk request was not made (due to valid cookie / s2s_error), we do not include risk_rtt in block / page_requested activities.